.travis.yml

```diff
@@ -5,5 +5,5 @@ jdk:
   - oraclejdk8
 
 sbt_args: -no-colors
-script: sbt ++$TRAVIS_SCALA_VERSION scalastyle clean coverage test unidoc
+script: sbt scalastyle clean coverage test unidoc "project unicorn-core" +test
```

Comment:
I did, the problem is that it tries to build `unicorn-play` with 2.12 then (which won't work as there is no play-slick for 2.12).

I am actually using `+test` but only narrowed it's scope to `unicorn-play`